functions/paperChannelTenders/src/dao/pn-tender-dao.ts

```diff
@@ -5,13 +5,13 @@ import { toPageMapper } from '../utils/mappers';
 import { PaperChannelTender } from '../types/dynamo-types';
 
 
-export const findTenders = async (page: number, size: number, from?: string, to?: string) => {
+export const findTenders = async (page: number, size: number, from?: Date, to?: Date) => {
   let scanInput = {
```

Comment:
Il codice può essere semplificato eliminando l'if

const defaultFrom = new Date("1970").toISOString();
const now = new Date().toISOString();

const scanInput: ScanInput = {
    TableName: PN_TENDER_TABLE_NAME,
    FilterExpression: "activationDate <= :to AND activationDate >= :from",
    ExpressionAttributeValues: {
        ":from": { S: from ? from.toISOString() : defaultFrom },
        ":to": { S: to ? to.toISOString() : now },
    },
};